app/src/js/components/button.jsx

```diff
@@ -1,21 +1,42 @@
 'use strict';
 
 const { React, Base } = require('./base');
+const ColorPicker = require('react-color');
+const sweetAlert = require('sweetalert');
+
+module.exports = class Component extends Base {
```

Comment:
I'd prefer to write the class and then export it by name below. Just a convention I use that I want to stick with.